### PR TITLE
create ref-tag creation script

### PR DESCRIPTION
**Pull request by maggu2810**
Create tag to reference a specified commit with a version modifier assigned.

### CODE REVIEW

**Comment by kaikreuzer** [comment]
Thanks! Shall I have a try with commit 27287e1a717d3eef3707449e00231acce712b9d5 as a "b1"?

**Comment by maggu2810** [comment]
:+1:

**Comment by hakan42** [comment]
Nice, thank you @maggu2810 :smile:

**Comment by maggu2810** [comment]
related to: #612

File: tools/ref-tag/create.sh [hunk 1]
```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+
+MY_CMD="${0}"
+MY_CMD_ABS="$(readlink -e "${MY_CMD}")"
```
Comment (kaikreuzer) [inline tools/ref-tag/create.sh]: I get a
```
readlink: illegal option -- e
```
since the -e option does not exist on my Mac. Any advice?

Comment (maggu2810) [inline tools/ref-tag/create.sh]: We could replace this line e.g. using
```bash
cd "$(dirname "${MY_CMD}")"
MY_CMD_ABS="${PWD}"
cd "${OLDPWD}"
```

Comment (maggu2810) [inline tools/ref-tag/create.sh]: I will modify the script in a branch, that you could test locally on your MAC. Okay? I tested with linux only.

Comment (kaikreuzer) [inline tools/ref-tag/create.sh]: ok!

Comment (maggu2810) [inline tools/ref-tag/create.sh]: https://github.com/maggu2810/smarthome/tree/ref-tag

File: tools/ref-tag/create.sh [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+if [ -z "${COMMIT_ID}" ]; then
+  die "commit id missing"
+fi
+
+#
+# Give the information to the user and wait for accept.
+#
+
+log "version postfix: ${VERSION_POSTFIX}"
+log "commid id: ${COMMIT_ID}"
```
Comment (kaikreuzer) [inline tools/ref-tag/create.sh]: typo: commid

**Pull request merged**